server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/processor/AbstractDlmPreprocessor.java

```diff
@@ -130,11 +139,15 @@ public Map<DataArchiveTableConfig, String> getDataArchiveSqls(Database database,
     private String generateTestSql(Database database, DataArchiveTableConfig table, List<OffsetConfig> variables) {
         try {
             DialectType dbType = database.getDataSource().getDialectType();
-            if (!dbType.isOracle() && !dbType.isMysql()) {
+            if (!dbType.isOracle() && !dbType.isMysql() && !dbType.isPostgreSql()) {
                 throw new UnsupportedException();
             }
             SqlBuilder sqlBuilder = dbType.isMysql() ? new MySQLSqlBuilder() : new OracleSqlBuilder();
-            sqlBuilder.append("SELECT 1 FROM ").identifier(database.getName(), table.getTableName());
+            if (dbType.isPostgreSql()) {
+                sqlBuilder.append("SELECT 1 FROM ").identifier("public", table.getTableName());
```

Comment:
why hardcode use 'public' schema for postgres?

Reply:
In the Postgres database, the public schema serves as the repository for user-defined data objects. In contrast, other schemas such as pg_catalog schema contain metadata related to the database system, and the information_schema holds information regarding the entire database. Here, as we anticipate presenting user-defined data objects, we employ the public schema.

Reply:
The hard-coded part has been removed.